src/zulip.rs

```diff
@@ -24,12 +24,29 @@ use axum::Json;
 use axum::extract::State;
 use axum::extract::rejection::JsonRejection;
 use axum::response::IntoResponse;
+use commands::BackportArgs;
+use octocrab::Octocrab;
 use rust_team_data::v1::{TeamKind, TeamMember};
 use std::cmp::Reverse;
 use std::fmt::Write as _;
 use std::sync::Arc;
 use subtle::ConstantTimeEq;
-use tracing as log;
+use tracing::log;
+
+const BACKPORT_APPROVED: &str = "
```

Comment:
Could you please turn these into functions that will receive the formatting arguments as parameters, and return a `format!`ed `String`? That way we'll be sure that we passed all the necessary parameters at compile-time.